dist/typings/proteus.d.ts

```diff
@@ -142,7 +142,7 @@ export declare module keys {
     static new(identity_key: keys.IdentityKey, prekey: keys.PreKey): keys.PreKeyBundle;
     serialise(): ArrayBuffer;
     serialised_json(): Object;
-    static signed(identity_key: keys.IdentityKey, prekey: keys.PreKey): keys.PreKeyBundle;
+    static signed(identity_key: keys.IdentityKeyPair, prekey: keys.PreKey): keys.PreKeyBundle;
```

Comment:
**Very good catch!** Please also rename `identity_key` to `identity_pair` to match the current implementation: https://github.com/wireapp/proteus.js/blob/v5.1.0/src/proteus/keys/PreKeyBundle.js#L72